.travis.yml

```diff
@@ -1,30 +1,12 @@
-sudo: false
+sudo: required
 
-language: go
+services:
+  - docker
 
-go:
-  - 1.9
-  - master
+before_install:
+  - docker build -t acr-builder .
```

Comment:
Yeah, next step is push and we would have to include the kraterdev registry name in the tag. But that's beyond the scope of this change